docs/BR.md

```diff
@@ -1616,16 +1596,23 @@ f. nameConstraints (optional)
 >
 > \* Non-critical Name Constraints are an exception to RFC 5280 (4.2.1.10), however, they MAY be used until the Name Constraints extension is supported by Application Software Suppliers whose software is used by a substantial portion of Relying Parties worldwide.
 
-g. extkeyUsage (optional)
+g. extkeyUsage (optional/required)
 
-> For Subordinate CA Certificates to be Technically constrained in line with section 7.1.5, then either the value id-kp-serverAuth [RFC5280] or id-kp-clientAuth [RFC5280] or both values MUST be present\*\*.
+> For Cross Certificates that share a Subject Distinguished Name and Subject Public Key with a Root Certificate operated in accordance with these Requirements, this extension MAY be present. If present, this extension SHOULD NOT be marked critical. This extension MUST only contain usages for which the issuing CA has verified the Cross Certificate is authorized to assert. This extension MAY contain the anyExtendedKeyUsage [RFC5280] usage, if the Root Certificate(s) associated with this Cross Certificate are operated by the same organization as the issuing Root Certificate.
+>
+> For all other Subordinate CA Certificates, including Technically Constrained Subordinate CA Certificates:
+>
+> This extension MUST be present and SHOULD NOT be marked critical.
 >
-> Other values MAY be present.
+> For Subordinate CA Certificates that will be used to issue TLS certificates, then either, or both, of the values id-kp-serverAuth [RFC5280] or id-kp-clientAuth [RFC5280] MUST be present. The values id-kp-emailProtection [RFC5280], id-kp-codeSigning [RFC5280], id-kp-timeStamping [RFC5280], id-kp-OCSPSigning [RFC5280], and anyExtendedKeyUsage [RFC5280] MUST NOT be present. Other values SHOULD NOT be present.
```

Comment:
I realize this is just carrying over from the current wording, but it would seem odd to me for a subCA to have only the id-kp-clientAuth EKU, while issuing TLS certificates. I'm guessing this is a misunderstanding on my part, but would it be "better" if this read something like "For Subordinate CA Certificates that will be used to issue TLS certificates, then the value id-kp-serverAuth [RFC5280] MUST be present and the value id-kp-clientAuth [RFC5280] MAY be present."

Reply:
Should be fixed in https://github.com/sleevi/cabforum-docs/pull/10/commits/45208d90819a95affa7403ca87c8bf32ada0eaa7